### PR TITLE
Feature/iops 1621 location

### PR DESCRIPTION
**Pull request by digitalhealthmartin**
Note: amended system in example to use a base international system because of a continue error at validation that was not able to be resolved.

Error:
[examples › England-Location-NightingaleHospital-Example.xml › FHIR Validation: validation/src/common.js.ts#L62](https://github.com/NHSDigital/NHSEngland-FHIR-ImplementationGuide/commit/040710014af3201eb6870109ef6b5b89ceb7b345#annotation_11918376743)
expect(received).toBeFalsy()

Received: "WARNING None of the codings provided are in the value set 'V3 Value SetServiceDeliveryLocationRoleType' (http://terminology.hl7.org/ValueSet/v3-ServiceDeliveryLocationRoleType|2014-03-26), and a coding should come from this value set unless it has no suitable code (note that the validator cannot judge what is suitable) (codes = http://terminology.hl7.org/ValueSet/v3-ServiceDeliveryLocationRoleType#HOSP) [ Location - Location.type[0]] [ Location - Line 1, Col 796]

### CODE REVIEW

File: CodeSystem/CodeSystem-England-OrganisationRole.xml [hunk 1]
```diff
@@ -0,0 +1,608 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (KazeemHamzat) [inline CodeSystem/CodeSystem-England-OrganisationRole.xml:1]: This Codesystem is a related asset to England-Organization profile but its needed here for validation. There is need to ensure they have the same content. Particularly the ID

File: StructureDefinition/Extension-England-ODSOrganizationRole.xml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+	  <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="An Extension to describe what an Organization does and whether this is the primary role of the Organization." />
+  <purpose value="An Extension to describe what an Organization does and whether this is the primary role of the Organization." />
```
Comment (KazeemHamzat) [inline StructureDefinition/Extension-England-ODSOrganizationRole.xml:21]: I will say the rule of no repetition in the NHSE CTA Dev. and guide may not allow duplicating content of description and purpose. Better to leave out purpose

File: StructureDefinition/Extension-England-ODSOrganizationRoleActivePeriod.xml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="Active period of an organization role." />
+  <purpose value="Active period of an organization role." />
```
Comment (KazeemHamzat) [inline StructureDefinition/Extension-England-ODSOrganizationRoleActivePeriod.xml:21]: Duplicate content for description and purpose

**Review by KazeemHamzat — CHANGES_REQUESTED** [review]

**Comment by digitalhealthmartin** [comment]
Removed unrelated files.

File: valusets/ValueSet-England-LocationType.xml [hunk 4]
```diff
@@ -0,0 +1,111 @@
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="England-LocationType" />
+    <url value="https://fhir.nhs.uk/ValueSet/England-LocationType" />
+    <version value="1.0.0" />
+    <name value="EnglandLocationType" />
+	<title value="England Location Type" />
+    <status value="draft" />
+    <description value="A ValueSet to identify the roles (type) of location." />
+    <compose>
```
Comment (SBajanki1) [inline valusets/ValueSet-England-LocationType.xml:9]: Purpose is missing

File: StructureDefinition/Extension-England-ODSOrganizationRole.xml [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  <copyright value="Copyright © 2018 Health and Social Care Information Centre. NHS Digital is the trading name of the Health and Social Care Information Centre." />
```
Comment (SheenaSharma) [inline StructureDefinition/Extension-England-ODSOrganizationRole.xml:22]: Copyright text needs to be changed.

**Comment by digitalhealthmartin** [comment]
**OLD PR. DO NOT MERGE. CLOSING PR.**

**Pull request closed without merging**